src/imitation/algorithms/adversarial.py

```diff
@@ -40,13 +41,13 @@ class AdversarialTrainer(base.BaseImitationAlgorithm):
 
     def __init__(
```

Comment:
Nit: if we're going to mess around with argument order, should we make all of these arguments kwarg-only? That way we can do that without any risk of silently breaking existing code.
(the breakage here is not going to be silent anyway, which is good, but perhaps changing this now gives us more flexibility in the future)

Reply:
Good point, I generally prefer kwargs anyway for functions with many arguments, I'll make that mandatory everywhere.